yt/visualization/plot_modifications.py

```diff
@@ -47,6 +47,9 @@ def _verify_geometry(func):
     @wraps(func)
     def _check_geometry(self, plot):
         geom = plot.data.ds.coordinates.name
+        # append dimensionality info to cylindrical geometry
+        if geom is "cylindrical":
```

Comment:
This should be `==` comparison. See e.g. https://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce